ratis-common/src/main/java/org/apache/ratis/protocol/DataStreamPacketHeader.java

```diff
@@ -19,29 +19,17 @@
 package org.apache.ratis.protocol;
 
 import org.apache.ratis.datastream.impl.DataStreamPacketImpl;
+import org.apache.ratis.proto.RaftProtos.DataStreamPacketHeaderProto.Type;
 import org.apache.ratis.util.SizeInBytes;
 
-import java.util.function.LongSupplier;
-
 /** The header format is streamId, streamOffset, dataLength. */
 public class DataStreamPacketHeader extends DataStreamPacketImpl {
-  private static final SizeInBytes SIZE = SizeInBytes.valueOf(24);
-
-  public static int getSize() {
-    return SIZE.getSizeInt();
-  }
-
-  public static DataStreamPacketHeader read(LongSupplier readLong, int readableBytes) {
-    if (readableBytes < getSize()) {
-      return null;
-    }
-    return new DataStreamPacketHeader(readLong.getAsLong(), readLong.getAsLong(), readLong.getAsLong());
-  }
+  private static final SizeInBytes SIZE_OF_HEADER_LEN = SizeInBytes.valueOf(4);
```

Comment:
why header size is 4 now?

Reply:
It's not header size, it's size of header size. For example, if header size is 24, we only need an int to store 24, and the size of int is 4.

Reply:
thanks for the clarification!